### PR TITLE
Cgroup parent

### PR DESCRIPTION
**Pull request by miaofenk**
Hi @jangaraj ,

I did a few code changes to make the cgroup parent branch supports containers under K8S environment. Would you please review my code changes and decide whether to take the pull request?

Thanks for your time!

### CODE REVIEW

**Comment by jangaraj** [comment]
👍 Which Kubernetes version has been used for your testing?

**Comment by miaofenk** [comment]
Tested using Kubernetes version:1.8.3.

**Comment by jangaraj** [comment]
Cc: @baoh0308 - version with `zbx_module_docker_get_cgroup_parent`

File: src/modules/zabbix_module_docker/zabbix_module_docker.c [hunk 1]
```diff
@@ -738,10 +817,19 @@ int     zbx_docker_dir_detect()
         {
             if ((strstr(path, "cpuset cgroup")) != NULL)
             {
+
+                #ifdef ZBX_VERSION_PRE_3_0_17
```
[the remark this answers is not in the document]
Comment (miaofenk) [reply src/modules/zabbix_module_docker/zabbix_module_docker.c]: I defined the marco when building with gcc using -D. The built module supported version depends on the compile options.

**Comment by jangaraj** [comment]
I'm merging this one into the feature branch. TODO:
- https://github.com/monitoringartist/zabbix-docker-monitoring/issues/89 still it needs some documentation how to compile it for 3.0.17- versions
- https://github.com/monitoringartist/zabbix-docker-monitoring/issues/85 I hope it will solve also AWS ECS issue
- (debug) logs cleaning (`zabbix_log(LOG_LEVEL_DEBUG, "zbx_module_update_driver: Debug 3.0");`)

Tested:
- CentOS 7 + Docker 18.05.0-ce
- Kubernetes 1.8.3

I'll be happy to receive any (un)sucessfull reports for other orchestratation tools (maybe @baoh0308 will be able to test it as well).

Anyway, thank you @miaofenk.

**Comment by baoh0308** [comment]
@jangaraj 
Thank you for quick test and merge.

@baoh0308

**Pull request merged**